contents/docs/libraries/python/index.mdx

```diff
@@ -220,6 +220,41 @@ posthog.get_feature_flag('beta-feature', 'distinct id', groups={'organisation':
 # returns string or None
 ```
 
+**Default overridden properties**
+
+Before posthog-python v3.0, by default we added geoip properties to all incoming events, and for feature flag evaluation, based on the IP address of the request. This isn't ideal since it uses your server IP address, rather than the user's, leading to incorrect location resolution.
```

Comment:
```suggestion
Before posthog-python v3.0, we added GeoIP properties to all incoming events by default. We also added them for feature flag evaluation, based on the IP address of the request. This isn't ideal since they are created based on your server IP address, rather than the user's, leading to incorrect location resolution.
```